copyoffload for 2025 (#248)
NearNodeFlash/NearNodeFlash.github.io: docs/guides/user-containers/readme.md

    workflows will fail in the `Setup` state due to insufficient resources. See [SystemConfiguration
    Setup](#systemconfiguration-setup).

### Rabbit Hostname Setup

> On each compute node there must be a file named `/etc/local-rabbit.conf` that contains the hostname of the compute's matching rabbit node. This will be used by compute node applications to communicate with the server in the user container. This is generally used for the non-MPI case.

Reviewer: ```suggestion
On each compute node there must be a file named `/etc/local-rabbit.conf` that contains the hostname of the compute's matching rabbit node. Compute node applications use this to communicate with the server in the user container. This is generally used for the non-MPI case.
```

This allows an application on the compute node to contact the user container running on its local
NNF node via these port numbers. The compute node must have proper routing to the NNF Node and needs
a generic way of contacting the NNF node. It is suggested than a DNS entry is provided via
a generic way of contacting the NNF node. It is suggested that a hostname entry is provided via
`/etc/hosts`, or similar.

For cases where one port is requested, the following can be used to contact the user container

> running on the NNF node (assuming a DNS entry for `local-rabbit` is provided via `/etc/hosts`).

Author: This is going to change, per the slack discussion.

Author: This has changed to using `/etc/local-rabbit.conf`.
